platform/android/export/export.cpp

```diff
@@ -2656,6 +2655,83 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	Error get_cl(const Ref<EditorExportPreset> &p_preset, const String &p_path, int p_flags, Vector<uint8_t> *clf) {
+
+		bool use_32_fb = p_preset->get("graphics/32_bits_framebuffer");
```

Comment:
Clean up these parameter names here and below: use snake_case and make them more complete